src/main/java/dwbh/api/domain/input/EmailGroupInput.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2019 Kaleidos Open Source SL
+ *
+ * This file is part of Don't Worry Be Happy (DWBH).
+ * DWBH is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * DWBH is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with DWBH.  If not, see <https://www.gnu.org/licenses/>
+ */
+package dwbh.api.domain.input;
+
+import java.util.UUID;
+
+/**
+ * UserGroup input. It contains the ids for a user and a group
+ *
+ * @since 0.1.0
+ */
+public class EmailGroupInput {
```

Comment:
There's a comment on what I think are the best naming possible for these inputs, we can't name a class by the  properties it contains, it should be used the operation they are used for. In this case `addUserGroup` -> `AddUserToGroupInput`.

Reply:
Done